src/cvsim/fit_curve.py

```diff
@@ -463,28 +466,11 @@ def __init__(
         self.alpha = alpha
         self.k_0 = k_0
 
-        # Contains only variables with a user-specified fixed value
-        self.fixed_vars = self._non_none_dict({
-            'reduction_potential': reduction_potential,
-            'diffusion_reactant': diffusion_reactant,
-            'diffusion_product': diffusion_product,
-            'alpha': alpha,
-            'k_0': k_0,
-        })
+        self.fixed_vars |= {'alpha': alpha, 'k_0': k_0}
```

Comment:
nit: maybe still have each param on a newline for readability

Reply:
done